library/core-test/src/main/java/com/github/mohammadsianaki/core_test/MainDispatcherRule.kt

```diff
@@ -0,0 +1,22 @@
+package com.github.mohammadsianaki.core_test
```

Comment:
Could you please unify this package name with the rest of the project?
I mean: com.github.mohammadsiamaki -> ir.kaaveh

P.S: You reminded me to change project package name to com.composenews 👍🏻

Reply:
Done✅